dist/radar_client.js

```diff
@@ -42,6 +42,193 @@ instance.Backoff = Backoff;
 
 module.exports = instance;
 },
+"lib/message.js": function(module, exports, require){var logger = require('minilog')('radar:message');
+
+var message_op_matrix = {
+  control: ['nameSync'],
+  message: ['publish', 'subscribe', 'sync', 'unsubscribe'],
+  presence: ['get', 'set', 'subscribe', 'sync', 'unsubscribe'],
+  status: ['get', 'set', 'subscribe', 'sync', 'unsubscribe'],
+  stream: ['get', 'push', 'subscribe', 'sync']
+};
+
+function Message (op, to) {
+  return {
+    op : op,
+    to : to
+  };
```

Comment:
this is also a bit weird. 

Instead of this we could have actual message objects here. So it is easy to ask:

```
var message = Message.build("subscribe", scope);
if(message.isValid()) {
   socket.send(message.serialize());
}
``